packages/api/test/functional/order.spec.js

```diff
@@ -347,7 +369,106 @@ test('PUT /orders/:id/close makes a seller closes an order successfully.', async
 	assert.isTrue(Bull.spy.called);
 });
 
-test('PUT /orders/:id/cancel returns an error when an unauthorized user attempts to cancel an order.', async ({
+test('PUT /orders/:id/close?orderType=service returns an error when an unauthorized buyer attempts to close a service order.', async ({
+	client,
+}) => {
+	const { user: buyerUser } = await createUser();
+	const { user: sellerUser } = await createUser();
+
+	const servicePurchased = await Factory.model('App/Models/Service').create({
+		user_id: sellerUser.id,
+	});
+
+	const serviceOrder = await Factory.model('App/Models/ServiceOrder').create({
+		service_id: servicePurchased.id,
+		user_id: buyerUser.id,
+	});
+
+	const response = await client
+		.put(`/orders/${serviceOrder.id}/close?orderType=service`)
+		.loginVia(buyerUser, 'jwt')
+		.send({ quantity: 3, unit_value: 100000 })
+		.end();
+
+	response.assertStatus(403);
+	response.assertJSONSubset(
+		errorPayload(errors.UNAUTHORIZED_ACCESS, antl('error.permission.unauthorizedAccess')),
+	);
+});
+
+test('PUT /orders/:id/close?orderType=service returns an error when a seller tries to close an order for a service with a non opened status.', async ({
+	client,
+}) => {
+	const { user: buyerUser } = await createUser();
+	const { user: sellerUser } = await createUser();
+
+	const servicePurchased = await Factory.model('App/Models/Service').create({
+		user_id: sellerUser.id,
+	});
+
+	const serviceOrder = await Factory.model('App/Models/ServiceOrder').create({
+		service_id: servicePurchased.id,
+		user_id: buyerUser.id,
+		status: serviceOrderStatuses.CLOSED,
+	});
+
+	const response = await client
+		.put(`/orders/${serviceOrder.id}/close?orderType=service`)
+		.loginVia(sellerUser, 'jwt')
+		.send({ quantity: 3, unit_value: 100000 })
+		.end();
+
+	response.assertStatus(400);
+	response.assertJSONSubset(
+		errorPayload(
+			errors.STATUS_NO_ALLOWED_FOR_OPERATION,
+			antl('error.operation.statusNoAllowedForOperation', {
+				op: 'CLOSE ORDER',
+				status: serviceOrder.status,
+			}),
+		),
+	);
+});
+
+test('PUT /orders/:id/close?orderType=service makes a seller closes a service order successfully.', async ({
+	client,
+	assert,
+}) => {
+	await Bull.reset();
+
+	const { user: buyerUser } = await createUser();
+	const { user: sellerUser } = await createUser();
+
+	const servicePurchased = await Factory.model('App/Models/Service').create({
+		user_id: sellerUser.id,
+	});
+
+	const serviceOrder = await Factory.model('App/Models/ServiceOrder').create({
+		service_id: servicePurchased.id,
+		user_id: buyerUser.id,
+	});
+
+	const response = await client
+		.put(`/orders/${serviceOrder.id}/close?orderType=service`)
+		.loginVia(sellerUser, 'jwt')
+		.send({ quantity: 3, unit_value: 100000 })
+		.end();
+
+	const orderClosed = await ServiceOrder.findOrFail(response.body.id);
```

Comment:
Pode usar o `.reload()` no model já criado anteriormente aqui para poupar trabalho
```suggestion
	await serviceOrder.reload();
```